daisy_workflows/build-publish/partner/cos_81_lts.publish.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Name": "cos-81-lts",
+  {{if eq .environment "oslogin-staging" -}}
+  "WorkProject": "oslogin-staging-images",
```

Comment:
Shouldn't this be oslogin-staging-project? Or did you set up a new project?

Reply:
No, this is just the env variable in the template. The publish templates are using oslogin-staging-project.

Reply:
Thanks!

Reply:
Oh yeah ok sorry- wrong project :)